src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,73 @@
+package seedu.addressbook.ui;
+
+import java.util.List;
+
+public class Formatter {
+
+    /** A decorative prefix added to the beginning of lines printed by AddressBook */
+    private static final String LINE_PREFIX = "|| ";
+
+    /** A platform independent line separator. */
+    private static final String LS = System.lineSeparator();
+
+    /** Offset required to convert between 1-indexing and 0-indexing.  */
+    public static final int DISPLAYED_INDEX_OFFSET = 1;
+
+    /** Format of indexed list item */
+    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+
+    /** Format of a comment input line. Comment lines are silently consumed when reading user input. */
+    private static final String COMMENT_LINE_FORMAT_REGEX = "#.*";
+
+    /**
+     * Returns true if the user input line should be ignored.
+     * Input should be ignored if it is parsed as a comment, is only whitespace, or is empty.
+     *
+     * @param rawInputLine full raw user input line.
+     * @return true if the entire user input line should be ignored.
+     */
+    public boolean shouldIgnore(String rawInputLine) {
+        return rawInputLine.trim().isEmpty() || isCommentLine(rawInputLine);
+    }
+
+    public String formatMessageShowToUser(String m) {
```

Comment:
Missing header comments for most of the methods. All non trivial methods should have javadoc format header comments.